libs/cms/src/lib/search/importers/supportQNA.service.ts

```diff
@@ -30,17 +30,19 @@ export class SupportQNASyncService implements CmsSyncProvider<ISupportQna> {
     return entries.reduce(
       (processedEntries: ISupportQna[], entry: Entry<any>) => {
         if (this.validateArticle(entry)) {
-          // We know that relatedLinks can contain circular references and that is dealt with during the mapping so we ignore it during the circularity check here
-          if (
-            !isCircular({
-              ...entry,
-              fields: { ...entry.fields, relatedLinks: [] },
-            })
-          ) {
-            processedEntries.push(entry)
-          } else {
-            logger.warn('Circular reference found in question', {
-              id: entry.sys.id,
+          try {
+            const mappedEntry = mapSupportQNA(entry)
+            if (!isCircular(mappedEntry)) {
+              processedEntries.push(entry)
```

Comment:
Is this really supposed to be the non-mapped `entry`?
```suggestion
              processedEntries.push(mappedEntry)
```